app/models/service_template_transformation_plan_task.rb

```diff
@@ -34,6 +34,67 @@ def task_active
     vm_resource.update_attributes(:status => ServiceResource::STATUS_ACTIVE)
   end
 
+  def conversion_host
+    Host.find_by(:id => options["transformation_host_id"])
+  end
+
+  def v2v_log
+    host = conversion_host
+    if host.nil?
+      msg = "Conversion host was not found: ID [#{options["transformation_host_id"]}]. Download of v2v log aborted."
+      _log.error(msg)
+      raise MiqException::Error, msg
+    end
+
+    userid, password = host.auth_user_pwd(:default)
```

Comment:
This should have `:remote` as the parameter which will fallback to `:default` if no "Remote" credentials are configured.

Example: https://github.com/ManageIQ/manageiq/blob/master/app/models/host.rb#L1024